recipes/diffpy.srfit/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "diffpy.srfit" %}
+{% set version = "3.1.0rc0" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/d/diffpy_srfit/diffpy_srfit-{{ version }}.tar.gz
+  sha256: 04b4e7658f39c837c07b4679ba23ce8fefb6d856fb8f0ba5e5fdbfe4914a815f
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - setuptools-git-versioning >=2.0
+    - pip
+
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - matplotlib-base
+    - scipy
+
+test: 
+  requires:
+    - python {{ python_min }} 
+    - pip
+    - pytest
+    - sympy
+  source_files:
+    - tests
+  commands:
+    - pip check
+    - pip list
+    - pytest
+
+about:
+  home: https://github.com/diffpy/diffpy.srfit/
+  license: Free To Use But Restricted
+  license_file: LICENSE.rst
```

Comment:
Custom license make it very hard for us to accept b/c we are not lawyers and we cannot evaluate all the possible interpretations and consequences. Can ask upstream to if they can use a well-known OSI approved license instead?

Reply:
I'm not a lawyer,  and sometimes recipes get merged without the proper license care. If you write "this is 100% compatible with bsd3" I would not mind merging it as-is. I will refer to your comment though if any questions arise. Is that ok?

PS: if you can get upstream to move to bsd3 it would be awesome!